core/build.gradle

```diff
@@ -1,5 +1,6 @@
 
 apply plugin: 'java'
+apply plugin: 'maven'
```

Comment:
is this really needed here? imho this is only required at app (only here we define upload tasks)